src/player.cpp

```diff
@@ -41,6 +41,7 @@
 #include <bitset>
 
 #include <fstream>
+#include <cmath>
```

Comment:
I recall cmath causing problems in the past.  Will test this anyway but not in the multimerge.